students/Mintae_K/lesson04/mailroom2.py

```diff
@@ -0,0 +1,69 @@
+# Sending a Thank you
+def sendingthank(donors):
+    f_name = input('Please type the Full Name or list> ')
+    while f_name == 'list':
+        print(' ')
+        for x in donors:
+            print(x)
+        print(' ')
+        f_name = input('Please type the Full Name or list> ')
```

Comment:
move this to line 5 (right after while loop) and you can remove line 3.

Reply:
Can't remove line 3. Because then 'while' loop will fail since it is testing for 'f_name == 'list'. If line 3 is removed, f_name is undefined.